requirements/edx/github.txt

```diff
@@ -74,7 +74,7 @@ git+https://github.com/edx/lettuce.git@0.2.20.002#egg=lettuce==0.2.20.002
 -e git+https://github.com/edx/event-tracking.git@0.2.1#egg=event-tracking==0.2.1
 -e git+https://github.com/edx/django-splash.git@v0.2#egg=django-splash==0.2
 -e git+https://github.com/edx/acid-block.git@e46f9cda8a03e121a00c7e347084d142d22ebfb7#egg=acid-xblock
-git+https://github.com/edx/edx-ora2.git@1.4.3#egg=ora2==1.4.3
+git+https://github.com/open-craft/edx-ora2.git@jill/django-file-upload#egg=ora2==1.4.4
```

Comment:
@efischer19 Let me know when you've cut the new tag, and I'll update this requirement to use it. 

Once that's done and tests pass, is it ok for me to squash and merge?